### PR TITLE
Issue 30

### PR DESCRIPTION
**Pull request by HaruVON**
Added Sunk and Surrender testing. Added SquareTest.java, and relevant testing. Added comments to the relevant test files. #30

### CODE REVIEW

**Comment by HaruVON** [comment]
Added more function testing in BoardTest.java, finished function testing in GameTest.java, added documentation, fixed bug: issue #34.

**Comment by HaruVON** [comment]
Still need to add getShips, setShips, getAttacks, and setAttacks function testing in BoardTest.java.

**Comment by HaruVON** [comment]
Will probably do that Friday morning, so hold off on pull request until that is finished.

**Comment by dillanea** [comment]
Since one our methods of finding a surrender is counting that all there are three sinks, maybe add a test to make sure that exists. There being three 3 sinks?

Our ships each have a hit length of 2, 3, 4, so having a test to make sure that once a sunk status is given, all ships have a hit length equal to their actual length

File: src/test/java/cs361/battleships/models/SquareTest.java [hunk 1]
```diff
@@ -7,4 +7,20 @@
...
+
+        square1.setRow(1);
+        square1.setColumn('A');
+        assertTrue(square1.getRow() == 1);
+        assertTrue(square1.getColumn() == 'A');
+
+        // Test: Override constructor, getRow, getColumn
+        Square square = new Square(1, 'A');
+        assertTrue(square.getRow() == 1);
+        assertTrue(square.getColumn() == 'A');
```
Comment (dillanea) [inline src/test/java/cs361/battleships/models/SquareTest.java]: I would add an assert false statement that would test cases that should fail. Example column != B

**Comment by HaruVON** [comment]
Added two new commits to the issue-30 branch. Covers assertFalse statements in SquareTest.java, testing checkGame, setAttacks, getAttacks, setShips, and getShips in BoardTest.java. Need a review.
#30

**Comment by dillanea** [comment]
Nice additions based off the suggestions we talked about. Looks great!

**Pull request merged**